Pbrp 8.1

wings110 opened the pull request:
goog job

rokibhasansagar committed:
Create config.yml

rokibhasansagar committed:
Update config.yml

Escape '<<'

rokibhasansagar committed:
Fix lunch & act-shell

rokibhasansagar committed:
Use repo sync

rokibhasansagar committed:
Update config.yml

Derp fix

rokibhasansagar committed:
No DT

rokibhasansagar committed:
Local manifest hack

rokibhasansagar committed:
Fix hack

rokibhasansagar committed:
Use roomservice for dt clone

rokibhasansagar committed:
Try minimal 8.0 manifest

rokibhasansagar committed:
Update config.yml

rokibhasansagar committed:
Use my script

rokibhasansagar committed:
Update omni_hero2qltechn.mk

rokibhasansagar committed:
Update config.yml

rokibhasansagar committed:
Fix Build on Oreo (#1)

* 1/3: Fix dt.img build on Oreo

* 2/3: Fix dt.img build on Oreo

* 3/3: Fix dt.img build on Oreo

rokibhasansagar committed:
Update omni_hero2qltechn.mk

rokibhasansagar committed:
Update bootimg.mk

rokibhasansagar committed:
Add cryptfs_hw repository from Lineage OS

[skip ci]

rokibhasansagar committed:
Crypto changes

rokibhasansagar committed:
Update omni.dependencies

rokibhasansagar committed:
Update omni.dependencies

[skip ci]

rokibhasansagar committed:
Update BoardConfig.mk

Minimal Crypto

rokibhasansagar commented:
@wings110, changes are yet not finished, still in test phase.
So @travismills82, don't merge it.

rokibhasansagar committed:
Correct TARGET_CRYPTFS_HW_PATH

[skip ci]

rokibhasansagar committed:
Fix TARGET_CRYPTFS_HW_PATH repo

rokibhasansagar committed:
Update omni.dependencies

Use TeamWin variant of cryptfs_hw
[skip ci]

rokibhasansagar committed:
BoardConfig: Change TARGET_CRYPTFS_HW_PATH

rokibhasansagar commented:
This modified repo tree is on "trial-and-error" method.
And it is not for Pure TWRP, rather than for other custom recovery.
So, it is better to close it for now.
When builds will be successful, new PR from a cleaner branch can be added.

Regards ✌️

rokibhasansagar committed:
Derp fix dependencies

rokibhasansagar committed:
BoardConfig: Remove TARGET_CRYPTFS_HW_PATH for now

rokibhasansagar committed:
Update init_hero2qltechn.cpp

"libinit" is simplified in android-8.1
Moved "vendor_init" into "property_services"

rokibhasansagar committed:
Update Android.mk

Broaden LOCAL_C_INCLUDES for libinit

rokibhasansagar committed:
Adapt new init_hero2qltechn.cpp

Signed-off-by: rokibhasansagar <rokibhasansagar2014@outlook.com>
